ballerina/listener.bal

```diff
@@ -72,6 +77,11 @@ public class Listener {
             if result is error {
                 return error Error("Error occurred while attaching the GraphiQL endpoint", result);
             }
+            if graphiql.printUrl {
+                string sanitizedPath = getSanitizedPath(graphiql.path);
+                string graphiqlUrl = string `http://${self.host}:${self.port}/${sanitizedPath}`;
+                io:println(string `GraphiQL client ready at ${graphiqlUrl}`);
```

Comment:
This should be printed inside the `'start` method, not in the attach method. 

Why do we append the `http` manually? What happens when we use SSL for the listener?

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-graphql/pull/1524/commits/8cd94860ec2d9d84fb8ec4ef27ecefcad27dd3b8